app/config.py

```diff
@@ -638,19 +638,22 @@ def read_webhook_enabled_user_ids() -> Optional[List[int]]:
 EVENT_LISTENER_DB_URI = os.environ.get("EVENT_LISTENER_DB_URI", DB_URI)
 
 
-def read_partner_domains() -> dict[int, str]:
-    partner_domains_dict = get_env_dict("PARTNER_DOMAINS")
-    if len(partner_domains_dict) == 0:
+def read_partner_dict(var: str) -> dict[int, str]:
```

Comment:
I would rename this method to `read_config_dict` so that it can be used in other places that have a similar need.

Reply:
This method uses the `get_env_dict`, which is generic, and adapts it to reading partner config, which will always have an integer as the key, and the value as string.

Reply:
ah true 👍